test_harness.py

```diff
@@ -94,7 +103,7 @@ def _run_stratis_cli_cert(namespace, unittest_args):
         + ["-v"]
         + unittest_args
     )
-    _run_command(3, command)
+    _run_command(namespace, 3, command)
```

Comment:
Do not pass ```namespace``` to ```_run_command```. You don't need to since you've configured the logger in the main method and the logging module will do the rest.